client/my-sites/media-library/media-date-range.scss

```diff
@@ -0,0 +1,125 @@
+.media-library__date-range {
+	position: relative;
+	display: flex;
+	align-items: baseline;
+}
+
+.media-library__date-range-span {
+	margin: 0 8px 0 0;
+	font-size: 0.9em;
+	color: #666;
```

Comment:
Please change all hex values to a corresponding sass variable, including #fff and #000. We are making some big changes in terms of color usage soon. Thanks!

Reply:
Agreed. Will do. Thanks